easybuild/easyconfigs/m/Molgenis-CmdLineAnnotator/Molgenis-CmdLineAnnotator.1.9.0-Java-1.8.0_45.eb

```diff
@@ -0,0 +1,35 @@
+name = 'CmdLineAnnotator'
+version = '1.9.0'
+namelower = name.lower()
+
+
+###  https://github.com/molgenis/molgenis/releases/download/v1.9.0/CmdLineAnnotator-1.9.0.jar
+
+
+homepage = ' https://github.com/molgenis/molgenis/releases/download/'
+
+description = """This distribution already contains several pipelines/protocols/parameter files which you can use 'out-of-the-box' to align and impute your NGS data using MOLGENIS Compute."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'JAR'
+
+java = 'Java'
+javaver = '1.8.0_45'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
+
+#
+# Example URL: https://github.com/molgenis/molgenis-pipelines/releases/download/2.1.1/NGS_DNA-2.1.1.tar.gz
```

Comment:
Yes, but then it should be https://github.com/molgenis/molgenis/.... instead of https://github.com/molgenis/molgenis-pipelines/...